stop.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 VARIABLE_SUM=$#
 CONTAINER_NAMES=$@
-CONTAINER_VALID_LIST="nginx lambdas content-server comms-server"
+CONTAINER_VALID_LIST="nginx lambdas content-server comms-server pow-auth-server"
```

Comment:
Is this OK to be added here now that it has been removed from everywhere else?

Is this to be able to delete the container even when is not on the compose file anymore?

Reply:
I get it. Pow server hasn't been removed from docker compose yet. Nevermind.

Reply:
That's it. I've tested all the changes so they are able to tier down POW server and not running it anymore!